js/src/Ice/Connection.js

```diff
@@ -44,35 +65,24 @@ export class IPConnectionInfo extends ConnectionInfo {
  **/
 export class TCPConnectionInfo extends IPConnectionInfo {
     constructor(
-        underlying,
-        incoming,
         adapterName,
         connectionId,
-        localAddress,
-        localPort,
-        remoteAddress,
-        remotePort,
-        rcvSize = 0,
+        localAddress = "",
```

Comment:
I would remove all the default param values. I don't think we ever use them in JS.